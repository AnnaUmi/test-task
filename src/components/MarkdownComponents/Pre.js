import React from 'react';

const Pre = props => <pre id="codes" {...props} />;

export default Pre;
