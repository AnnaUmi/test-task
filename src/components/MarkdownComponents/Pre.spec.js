import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Pre from './Pre';

describe('Pre', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Pre />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
