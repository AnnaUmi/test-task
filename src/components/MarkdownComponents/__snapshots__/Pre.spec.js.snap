// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pre should match snapshot 1`] = `
<pre
  id="codes"
/>
`;
